frontend/src/views/ddahs/index.tsx

```diff
@@ -83,6 +126,45 @@ export function AdminDdahsView(): React.ReactNode {
                 >
                     Approve DDAH
                 </ActionButton>
+                <ActionButton
+                    icon={FaTrash}
+                    onClick={confirmDDAHDeletion}
+                    disabled={selectedDdahIds.length === 0}
+                >
+                    Delete DDAH
+                </ActionButton>
+                <Modal
+                    show={ddahDeletionConfirmationVisible}
+                    onHide={() => {
+                        setDdahDeletionConfirmationVisible(false);
+                    }}
+                >
+                    <Modal.Header closeButton>
+                        <Modal.Title>Deleting Multiple DDAHs</Modal.Title>
+                    </Modal.Header>
+                    <Modal.Body>
+                        You are deleting all of the following{" "}
+                        {selectedDdahs?.length} DDAHs!
+                        <br />
```

Comment:
Don't use `<br />` elements. If you need padding, wrap things in a div and use, for example `className="mb-2"` to get a bottom margin.